README.md

```diff
@@ -2,58 +2,121 @@
 
 Setup all services needed for the SCS Status Page.
 
-## Secrets
+## Local deployment
 
-### Database
+The local deployment uses [`kind`](https://kind.sigs.k8s.io/) to set up a local k8s cluster to deploy to.
 
-Database credentials in [`kubernetes/environments/kind/database/database.env`](kubernetes/environments/kind/database/database.env) are purely used as example or local debug credentials.
+```bash
+kind create cluster --name status-page --wait 3m --config=kind-cluster-config.yaml
+# or
+make cluster
+```
 
-### Dex
+### Prerequisites
 
-Dex needs the GitHub Client ID and Client Secret in `kubernetes/environments/kind/dex/dex.env`:
+Please deploy the [`kubernetes/environments/kind/prerequisites/kustomization.yaml`](kubernetes/environments/kind/prerequisites/kustomization.yaml) to set up the local `kind` cluster to work with the deployment.
 
-```env
-GITHUB_CLIENT_ID=<client_id>
-GITHUB_CLIENT_SECRET=<client_secret>
+```bash
+kubectl --context kind-status-page apply -k kubernetes/environments/kind/prerequisites
 ```
 
-As well as a client config in [`kubernetes/environments/kind/dex/config.yaml`](kubernetes/environments/kind/dex/config.yaml):
+#### Persistent Volume Claims
+
+`kind` does not provide a way to create a persistent volume for persistent volume claims. To be able to satisfy volume claims, rancher's `local-path-storage` is deployed locally.
```

Comment:
`kind` already deploys `local-path-storage` out of the box, doesn't it? So there should not be any need to deploy it as custom "prerequisite"?

Reply:
When i started the deployment project, requesting a PV(C) kept hanging in `pending` state. Solution was to deploy the local path storage. `kind` now seems to work with their own provider. Removed the prerequisite.